mangaki/mangaki/templatetags/version.py

```diff
@@ -0,0 +1,18 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=False)
+def mangaki_version():
+    return settings.VERSION.split('+')[0]
+
+@register.simple_tag(takes_context=False)
+def mangaki_revision():
+    try:
```

Comment:
Could you provide an example?
Like `settings.VERSION = v0.2.devXX+hash.timestamp` will output hash.
Like `settings.VERSION = v0.5.3` will output ''. Is it expected behavior? I feel like I was lucky.

Reply:
What do you mean by providing an example? Adding `doctests` or documentation?
But yes, a custom tag will give no revision. If a SHA1 hash is available, it'll be used.

Reply:
Documentation.

Can't you realize? The day we will get to `v10`, the output will be `'1'`. Right? I think there is a problem.

Reply:
The day we'll get to `v10`, we'll have `VERSION = 'v10.0.3.devXX+ghash.timestamp`
If I split with `+` then take the last part and split by `.`, won't it be alright?
As for the `VERSION`, splitting by `+` and taking the first part will always give the full tag or the dirty tag, right?

Reply:
Let's say we have tag = 'v10.1'.
split('+'), take last part => 'v10.1'
split('.'), take first part => 'v10'
[:1] => Return '10'
Will this, as a link, work?

Reply:
Well, you're right, I should ensure that we have at least two items after `split('+')`.

Reply:
Can you write a test? 😄

Reply:
I will.